src/utils/computeSliderStyle.js

```diff
@@ -0,0 +1,35 @@
+import {merge, memoizeWith, identity, contains} from 'ramda';
+
+export default () => {
+    return memoizeWith(identity, (vertical, verticalHeight, tooltip) => {
+        let style = {
+            padding: '25px',
+        };
+
+        if (vertical) {
+            style = merge(style, {height: verticalHeight + 'px'});
```

Comment:
`style` is created inside the function scope, it's fine to consider it as mutable inside the function